service.subtitles.napisy24pl/resources/lib/NapisyUtils.py

```diff
@@ -145,7 +145,7 @@ def download(self, id):
         for file in files:
             if os.path.splitext(file)[1] in exts:
                 filename_dest = os.path.join(__temp__, file)
-                flag = xbmcvfs.copy(os.path.join(zip_filepath, file), filename_dest)
+                flag = xbmcvfs.copy(zip_filepath + '/' + file, filename_dest)
```

Comment:
consider adding a comment here to avoid the same request in the future (free to add on future version)